tests/tests_full_scan_engine.py

```diff
@@ -115,6 +115,7 @@ async def test_jackson_number_objects(self):
         aidb_res = aidb_engine.execute(aidb_query)
         # TODO: equality check should be implemented
         assert len(gt_res) == len(aidb_res)
+        assert sorted(gt_res) == sorted(aidb_res)
```

Comment:
I don't think this works across all engines with floating point values, does it? Also, there are other places that do full scans across the tests

Reply:
It works. I updated other full scan tests as well

Reply:
This approach is valid because both ground truth and AIDB results are retrieved from the same type of database. However, for queries involving user defined functions, the results are computed externally, which may lead to slight discrepancies in MySQL database.

Reply:
@ttt-77 do you have test in the new code for this?

Reply:
Yes. Using 'sorted(gt_res) == sorted(aidb_res)' in this function is expected to lead to an assertion error.
https://github.com/ddkang/aidb/blob/e64be7326fcde29b8878b8c328723dd763dcd7d0/tests/tests_full_scan_engine_udf.py#L343

Reply:
@zencodess ^

Reply:
Do we have an issue for this?

Reply:
We have updated the code to accommodate the difference caused by keeping a certain number of decimal places